docs/src/main/sphinx/release/release-430.md

```diff
@@ -27,17 +27,52 @@
 ## Ignite connector
 
 * Fix failure when a query contains `LIKE` with `ESCAPE`. ({issue}`19464`)
+* Improve reliability of connecting to the source database. ({issue}`19191`)
+
+## ClickHouse connector
+
+* Improve reliability of connecting to the source database. ({issue}`19191`)
+ 
+## Druid connector
+
+* Improve reliability of connecting to the source database. ({issue}`19191`)
+ 
+## Druid connector
+
+* Improve reliability of connecting to the source database. ({issue}`19191`)
 
 ## MariaDB connector
 
 * Add support for table statistics. ({issue}`19408`)
-
+* Improve reliability of connecting to the source database. ({issue}`19191`)
+ 
 ## MongoDB connector
 
 * Fix incorrect results when a query contains several `<>` or `NOT IN`
   predicates. ({issue}`19404`)
+* Improve reliability of connecting to the source database. ({issue}`19191`)
```

Comment:
MongoDB connector isn't JDBC based.

Reply:
Thanks for being 'sharp' here - I wasn't